Aquality.Selenium.Template/Aquality.Selenium.Template/Utilities/CustomImageComparator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Aquality.Selenium.Core.Configurations;
+using Aquality.Selenium.Core.Visualization;
+using System.Drawing;
+
+namespace Aquality.Selenium.Template.Utilities
+{
+    public static class CustomImageComparator
```

Comment:
Why do we want to have such class, what's the main point? Why can't we use aquality interfaces directly?
Is it about dynamic configuration?

Reply:
The code was updated